query/src/sql/exec/expression_builder.rs

```diff
@@ -39,7 +39,12 @@ impl<'a> ExpressionBuilder<'a> {
     }
 
     pub fn build_and_rename(&self, scalar: &Scalar, index: IndexType) -> Result<Expression> {
-        let expr = self.build(scalar)?;
+        let mut expr = self.build(scalar)?;
+        if let Expression::AggregateFunction { .. } = expr {
```

Comment:
If we want to deal with some other case in the future, `match` is better?

Reply:
I'm not sure if we'll add more special cases like `AggregateFunction`. Even if it does, https://github.com/datafuselabs/databend/blob/main/common/planners/src/plan_expression_chain.rs#L73 will prompt us to take care of it

Reply:
Will this work with `count(*) + 1`?

Reply:
Hold on a second, it's almost ready 🤣